### DISCUSSION TITLE
Any tweaking needed?

### DISCUSSION THREAD

**Discussion by Ergp3** [ALKindi-Wilagabrata/DELL-3421#2] [Q&A]
Hi! I wanted to know if there's any special process I would have to do to get this working or if the opencore guide suffices.

**Comment by alkindivv** [comment]
Hi, Yes SandyBridge and IvyBridge need some tweaking to get bigsur working on it
if you need help for installing hackintosh you can contact me on telegram or whatsapp +6281267041916

**Reply by Ergp3** [reply]
Ok, thanks

**Comment by alkindivv** [comment]
Hi, Yes SandyBridge and IvyBridge need some tweaking to get bigsur working
on it
if you need help for installing hackintosh you can contact me on telegram
or whatsapp +6281267041916
